From: "duncan.macleod" <duncan.macleod@ligo.org>
Subject: [PATCH] Fix setup for windows

conda-forge's librdkafka ships rdkafka.lib on Windows, not librdkafka.lib.
---
diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -11,10 +11,7 @@
 
 # On Un*x the library is linked as -lrdkafka,
 # while on windows we need the full librdkafka name.
-if platform.system() == 'Windows':
-    librdkafka_libname = 'librdkafka'
-else:
-    librdkafka_libname = 'rdkafka'
+librdkafka_libname = 'rdkafka'
 
 module = Extension(
     'confluent_kafka.cimpl',
